### ISSUE TITLE
parse error :class include Array

### ISSUE THREAD

**Issue by rambo-panda** [vuejs/eslint-plugin-vue#674]
**Tell us about your environment**	
* **ESLint version:**	5.8.0
* **eslint-plugin-vue version:**	5.0.0-beta.3
* **Node version:**	  10.3

**Please show your full configuration:**

```json
{
	"plugins": ["vue"],
	"extends": ["plugin:vue/essential"],
}
```

**What did you do?**	

```vue
<div v-for="item in array">
     <div v-bind:class="{abc: ['x', 'y'].includes(item)}">...</div>
</div>
```

**What did you expect to happen?**
I should be able to use **Array** syntax normally

**What actually happened?**

```
   28:53  error  'v-bind' directives require an attribute value         vue/valid-v-bind
   29:63  error  Parsing error: Unexpected end of expression            vue/no-parsing-error
   29:64  error  Parsing error: missing-whitespace-between-attributes   vue/no-parsing-error
   29:78  error  Parsing error: unexpected-character-in-attribute-name  vue/no-parsing-error
   29:81  error  Parsing error: unexpected-character-in-attribute-name  vue/no-parsing-error
   29:99  error  Parsing error: unexpected-character-in-attribute-name  vue/no-parsing-error
   30:30  error  Parsing error: unexpected-character-in-attribute-name  vue/no-parsing-error
```

**Comment by mysticatea** [comment]
Hi. thank you for the report.

But I couldn't reproduce it.
Would you make a repro case on [our online demo](https://mysticatea.github.io/vue-eslint-demo/#eJyNVU1zmzAQ/SseLmlnrGSmR0/aP9Hemh6EWECtkOhKOCaZ/Pc+ISPA+NCDPei9ZbX79oP3QrmKilPxHKjrjQz07cUeDs+VPk8P6fFwFrXjry+FhtVB24NkluNLcTXJRqW21UkZ6T1s32WpToefD5eH4+FhfPj1qK0yQ0X+U/Ty+QOvPz4+Pj+trsrP89PzUw6rOBY8GPLF6R0xWx94UMGx8ENPXJy+HAuEKCrNpIJ2dkIaCoFYMIWBE2KdUNKTqAhhsoyWPhMxcIGfbhZj1/WSSVhqxBuxW3Bb7Uytd4bWx3DHJEgbptd1DjMxgZ1BqA1dZrCicmiaa3bT2VAgcZYLguSF5MZvgJRIR11JvGX+0LgGjFaQdlJkRiF3GIVqIY6K2k2+tmQvo6p2A+bD5SZlugSWonRQRtp4U9gynjo9I7U0JrTshqbN0GDVjcfGuFKaG1Bbi2DvVVXbszS6Ssr2GWWcBxiL1xbt6HupsgSdvlAlJgjtgDoHWWZ3ll6FH7vSmRlx5W/kZUw2cSrIzDKloGgBEMfV+4x5MvVNQjBgH2uLQct2odVelIRGp1XfgxhsRfVyQKKYAiTRDSZooy0tHJNEdUuzgrysSdTaIonxBkXfy3WfDnbw8GtkSUvCVxB9uUBIidCE5JXs01XQCUEJ3zoOLWSdQKa/A0ZWjJpMQvCm0N7KdGWqXRh7cim/80BPGMqOMEXXYT9TZn77S7zaL7FEFPFIP6KRtJ0GeojS9OwgYNC0scMc1oPXthHTysO/rtf8dooW8DpJmAzWJfxv+FjK6JKYHa8JJk98RjC3Hj3mD3AM1tkpVRnkxkBXJKiukb//j7TmLRov2uKXgJskGgWLSbW0Jq9lzUH4O+StzrF6UTPxqkObNMz0XOslK+13ZIx/qrdYLfmdFRMafrPY12wq3TrVzEzthFmRGIzprpVJ9HVPy8XNtRtnMdm5cMOlb+AOVMbJPzsUI0Tr/lrjO3Ct5Iy1oTM78I7DDt/4vaGzdyC1v7nnPYYpft2BsZsAfhwLaID6ftdv6c10/DHGTVCkXXsspgWHnApsXyYqPv4B3Ernww==)?

**Comment by rambo-panda** [comment]
I'm terribly sorry！

The root cause is the reason for our data format。

Our template is directly given by the server to the corresponding template, our data engineer did a data filtering
